index.html

```diff
@@ -45,18 +44,20 @@ <h3>Fighters</h3>
                 <!--Found Me!-->
 
                 <h3>Haiku</h3>
-                Butterflies are so cool </br>
-                in the big, huge, green forest </br>
-                they fly up so high. </br>
+                <p>
+                Butterflies are so cool <br>
+                in the big, huge, green forest <br>
+                they fly up so high. <br>
                 <cite>by <span class="author">Unknown</span></cite>
+                </p>

```

Comment:
The whole haiku and author line was to be inside a `<p>`
`<br>` vs `</br>` again